### DISCUSSION TITLE
How to use OpenDCT for streaming services?

### DISCUSSION THREAD

**Discussion by FoggySageTV** [OpenSageTV/Community#5] [OpenDCT]
I know this is quite an open topic ...
Is there a concise guide using OpenDCT for the various streaming services?
John

**Comment by Ugadata** [comment]
That would depend on what "streaming services" you would like to get using OpenDCT.

I am not aware that the general content of Amazon, Hulu, Netflix, etc can be accessed in SageTV.

There are some some cable TV replacement services available (Hulu Live, YouTube TV, Philo, Friendly TV, etc) that can be accessed using OpenDCT and Channels DVR.  There is also a project that emulates an HDHR Tuner that can used with OpenDCT as well.  The only downsides of Channels DVR is it is stereo only, and there are some channels that Channels can't access either.


Digression - Several years ago (mid 00's) Sage had an option to connect to Hulu Desktop. But that went away a long time time ago.

There are some products that can download content from the various streaming providers and can be imported into Sage.  However many would use something else beside Sage for viewing downloaded content i.e. Emby, Jellyfish, etc.
